presto-mongodb/src/test/java/io/prestosql/plugin/mongodb/BaseMongoDistributedQueries.java

```diff
@@ -13,33 +13,21 @@
  */
```

Comment:
I would name this commit as: `Extract BaseMongoDistributedQueries`. Also please that we are removing test coverage here and restoring in subsuequent commit, and that is because that we won't to retain the file changes history.